Source/WebKit/UIProcess/WebPageProxy.h

```diff
@@ -3231,6 +3238,9 @@ class WebPageProxy final : public API::ObjectImpl<API::Object::Type::Page>, publ
 #endif
 
     RefPtr<WebPageProxy> m_pageToCloneSessionStorageFrom;
+
+    HashMap<WebCore::RegistrableDomain, UniqueRef<SubframePageProxy>>  m_subframePageProxyForDomain;
```

Comment:
I don't think these HashMaps have good names. It is a container but it sounds like a single SubframePageProxy.
Also you have an extra space between the type and the name.

For the name, I could see something like m_domainToSubframePageProxyMap.